docs/development-guide.md

```diff
@@ -46,7 +46,7 @@ it is **highly** encouraged to:
 * Set up a pre-commit hook to catch issues locally.
 
    ```console
-   pip install pre-commit==2.5.1
+   pip install pre-commit==2.20.0
```

Comment:
does it need to be versioned, why not use the latest?

Reply:
need not be, I put it without the version in that case.